CalculatorTests/CalculatorBrainTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by Wayne Rumble on 05/06/2017.
+//  Copyright © 2017 Wayne Rumble. All rights reserved.
+//
+
+import XCTest
+@testable import Calculator
+
+class CalculatorBrainTests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    func testConstants() {
+        let symbolsAndValues = ["π": Double.pi, "e": M_E]
```

Comment:
It depends. If there's only a few, I would duplicate.

If there's loads, you could consider moving the assert into a "helper" that gives a better message saying which one went wrong (if possible :grin:)